tests/BasicSearchTests.FunctionalTests.Core/Models/V2SearchResultEntry.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace BasicSearchTests.FunctionalTests.Core.Models
+{
+    public class V2SearchResultEntry
+    {
+        [JsonProperty(Required = Required.Always)]
+        public string Version;
+
+        [JsonProperty(Required = Required.Always)]
+        public string NormalizedVersion;
+
+        public string Title;
+
+        public string Description;
```

Comment:
Sure. I was following the pattern used for the `V3SearchResultEntry`, I will update that guy as well.